src/utils/getFormatter.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const path = require('path');
+const _ = require('lodash');
+
+const isPath = x => x.indexOf('/') > -1;
+
+const getNameFromObject = formatterObject => {
+  if (!formatterObject.name) {
+    throw new TypeError("Formatter configuration 'name' key is missing.");
+  }
+  return formatterObject.name;
+};
+
+const loadFormatter = formatterPath => {
+  /* eslint-disable import/no-dynamic-require */
+  try {
+    return require(formatterPath);
+  } catch (error) {
+    error.message = `Formatter failed to load: ${formatterPath}\nError:\n\n${error.message}`;
+    throw error;
+  }
+  /* eslint-enable */
+};
+
+const getFormatter = function(formatter) {
+  let formatterName = formatter || 'default';
+
+  if (_.isObject(formatter)) {
+    formatterName = getNameFromObject(formatter);
+
+    if (formatter.thirdParty) {
+      return loadFormatter(formatterName);
+    }
+  }
+
+  let prefix = 'src/formatters';
+
+  if (isPath(formatterName)) {
+    prefix = '';
+  }
+
+  return loadFormatter(path.resolve(prefix, formatterName));
+};
```

Comment:
This is pretty much the code from ESLint's `getFormatter`, but with a few changes. They do have a line in there where they replace backslashes with forward slashes for Windows users, but my assumption was that `path.resolve` would take care of this. I'm a bit noobish with node's libraries though, so can someone clarify here?